catroid/src/main/java/org/catrobat/catroid/content/actions/ThinkSayBubbleAction.java

```diff
@@ -47,14 +47,15 @@ protected void update(float delta) {
 			Log.d(getClass().getSimpleName(), "Formula interpretation for this specific Brick failed.", interpretationException);
 			return;
 		}
-		ShowBubbleActor showBubbleActor = new ShowBubbleActor(textToDisplay, sprite, type);
-
 		if (StageActivity.stageListener.getBubbleActorForSprite(sprite) != null) {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1794599/24807922/63874ce6-1bba-11e7-81d1-0cbec4301e2d.png)
According to Scratch, yes